recipes/revtok/meta.yaml

```diff
@@ -0,0 +1,40 @@
+# upstream did not publish any tags, and a performance-critical
+# fix appears after the last bump (for 0.0.3). This is what e.g.
+# torchtext uses upstream (installing through git), so we add a ".1"
+{% set version = "0.0.3.1" %}
```

Comment:
Reason for this is https://github.com/pytorch/text/blob/v0.11.0-rc3/requirements.txt#L11, and wanting not to leave out a potentially very substantial performance improvement from https://github.com/jekbradbury/revtok/pull/4 (the PR notes 100-fold improvement by interning strings in some cases), which is fact the only change after 0.0.3

Reply:
it might be better to patch, just to reference the pypi version but ok.

Reply:
You mean use 0.0.3 and carry a patch for https://github.com/jekbradbury/revtok/pull/4? I don't mind either way. That repo looks pretty dead otherwise (last commit 4.5 years ago), and I don't expect anyone to start using `micro` versions even if it does get revived, which is why I chose it.